src/sentry/api/base.py

```diff
@@ -577,17 +569,6 @@ def track_set_commits_local(self, request: Request, organization_id=None, projec
         )
 
 
-class PreventNumericSlugMixin:
```

Comment:
Originally, I only used a mixin because using a pattern directly on the slug regex field could not be feature-flagged